tree-sitter.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://tree-sitter.github.io/tree-sitter/assets/schemas/config.schema.json",
+  "grammars": [
+    {
+      "name": "zeek",
+      "camelcase": "Zeek",
+      "title": "Zeek",
+      "scope": "source.zeek",
+      "file-types": ["zeek"],
+      "injection-regex": "^zeek$",
+      "class-name": "TreeSitterZeek"
+    }
+  ],
+  "metadata": {
+    "version": "0.2.3",
+    "license": "MIT",
```

Comment:
This should also be BSD-3?

Reply:
Yes, definitely, thanks for catching it. I'll also fix the license key in `Cargo.toml`.